rpc/clients/typescript/package.json

```diff
@@ -37,8 +30,6 @@
     },
     "homepage": "https://github.com/0xProject/0x-mesh/rpc/clients/typescript/README.md",
     "dependencies": {
-        "@0x/assert": "^2.1.2",
-        "@0x/dev-utils": "^2.2.6",
```

Comment:
`@0x/dev-utils` on the other hand is only used in tests and can therefore be moved. ✅